apps/builder/app/builder/features/style-panel/sections/transforms/transform-and-perspective-origin.tsx

```diff
@@ -141,14 +135,28 @@ export const TransformAndPerspectiveOrigin = (
         properties={[property]}
       />
       <Flex gap="6">
-        <Grid css={{ gridTemplateColumns: "1fr 1fr" }} align="center" gapX="2">
+        <Grid css={{ gridTemplateColumns: "1fr 2fr" }} align="center" gapX="2">
           <PositionGrid
             selectedPosition={{ x: xInfo, y: yInfo }}
             onSelect={handlePositionGridChange}
           />
           <Flex gap="2" direction="column">
-            <Flex gap="2" align="center">
-              <Label>X</Label>
+            <Grid
+              gap="2"
+              align="center"
+              css={{ gridTemplateColumns: "auto 1fr" }}
+            >
+              <PropertyInlineLabel
+                label="X"
+                title={
+                  property === "transformOrigin" ? "Offset X" : "X Position"
```

Comment:
btw why is it not both Offset X Position X or X Offset and X Position ? I wonder what's the right way

Reply:
In MDN, they seem to use different wordings to represent these two
https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin#values

https://developer.mozilla.org/en-US/docs/Web/CSS/perspective-origin#values

Reply:
checked mdn, it uses x-offset and x-position, so why changing it and why not using humanizeString to get that value from data?

Reply:
Btw, we are not storing these things from the `mdn` right. To use it directly via `config`. Am i missing something, from where to use it 🤔. Except to keep them as strings like above.

Reply:
I thought you already had a config, but maybe I misremembered, you prob only have the long-form that combines property and position name

Reply:
Done 👍